infrastructure/dpladm/bin/dpladm-shared.source

```diff
@@ -76,6 +76,10 @@ function renderProfileTemplate {
   local primaryDomain=${8:-}
   # A space-seperated list of secondary domains (optional)
   local secondaryDomainsString=${9:-}
+  # The primary GO subdomain (optional)
+  local primaryGoSubDomain=${10:-}
+  # A space seperated list of secondary GO sub domains (optional)
+  local secondaryGoSubDomains=${11:-}
```

Comment:
What happens if there are no secondary domain strings?

Reply:
I see you'll then just have empty double quotes - right?

Reply:
yes